src/openms/source/CONCEPT/Constants.cpp

```diff
@@ -172,6 +172,8 @@ namespace OpenMS
       // User parameter name for precursor mz error in ppm
       const std::string FRAGMENT_ANNOTATION_USERPARAM = "fragment_annotation";
 
+      const std::string PSM_EXPLAINED_ION_CURRENT_USERPARAM = "PSMExlpainedIonCurrent";
```

Comment:
hmm looking at the other names I think: PSM_explained_ion_current would be more in line how we capitalize these user params